notebooks/ai-ml/HuggingFaceEmbeddingGenerationInSpark.ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<p>Please expand on this. This isn't an easy topic for a user to understand so the more explanation the better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>